### PR TITLE
Post migration automation

### PR DESCRIPTION
**Pull request by rahulippar**
Hi @go-davidng or Amit, can you please review and approve this pull request. This is task for unjoining and rejoining domain in case of clusterred mssql database migration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,15 @@ script uses an open source utility `rsync` with the some default commands.
...
-COS and restores the backup db.
+COS and restores the backup db. [Click here](db-migration/mysql/) for script.
+
+- [MSSQL](db-migration/mssql/post-migration/)
 
-[MySQL db migration](db-migration/mysql/)
+   In case of MSSQL clustred database migration using [Rackware Management Module (RMM)](https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-mssql-db-overview#rackware-management-module) there is known issue. When MSSQL migration is performed for machines with clustered node. User will not be able to login with domain credentials because of error "the trust relationship between this workstation and the primary domain failed". So even if you enter the correct domain credentials post migration, it does not allow to user to login. So as a simple solution to this problem is to login to the node machine using local admin account and explicitly unjoin node machine from domain and rejoin domain. Once this is done, user can login to the target machine using domain credentials. This process is automated in the form of script. [Click here](db-migration/msssql/post-migration) for script.
```
Comment (go-davidng) [inline README.md:38]: This is a post-script for clustered MSSQL migration with the [Rackware RMM tool](https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-mssql-db-overview#rackware-management-module]) that addresses domain joins for the new target MSSQL clusters.

File: db-migration/mssql/post-migration/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Overview
+In case of MSSQL clustred database migration using [Rackware Management Module (RMM)](https://cloud.ibm.com/docs/cloud-infrastructure?topic=cloud-infrastructure-mssql-db-overview#rackware-management-module) there is known issue. When MSSQL migration is performed for machines with clustered node. User will not be able to login with domain credentials because of error "the trust relationship between this workstation and the primary domain failed". So even if you enter the correct domain credentials post migration, it does not allow to user to login. So as a simple solution to this problem is to login to the node machine using local admin account and explicitly unjoin node machine from domain and rejoin domain. Once this is done, user can login to the target machine using domain credentials. This process is automated in the form of script. [Click here](db-migration/msssql/post-migration) for script.
```
Comment (go-davidng) [inline db-migration/mssql/post-migration/README.md:2]: When migrating a clustered MSSQL, the new target MSSQL clustered DB does not properly join the domain due to the trust relationship. This prevents login with the domain credentials.

The script helps resolves this by logging into the new clustered DBs with local admin credentials and explicitly unjoining and rejoining the domain.

**Review by go-davidng — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rahulippar) [reply README.md]: Changes are implemented.

**Review by go-davidng — APPROVED** [review]
looks good

**Pull request merged**